test/capybara_session_extensions.rb

```diff
@@ -3,24 +3,24 @@
 #
 #  = Integration High-Level Test Helpers
 #
-#  Methods in this class are available to all the integration tests.
+#  Methods in this class are available to Capybara integration tests.
 #
 #  login::   Create a session with a given user logged in.
 #  login!::  Same thing,but raise an error if it is unsuccessful.
 #
```

Comment:
Note that the `IntegrationExtensions` methods `login` and `login!` were moved to `SessionExtensions` for rails-dom-testing. They're always included together.